roaring64/bsi64.go

```diff
@@ -37,6 +39,23 @@ func NewDefaultBSI() *BSI {
 	return NewBSI(int64(0), int64(0))
 }
 
+func (b *BSI) RunOptimize() {
+	b.eBM.RunOptimize()
+	for i := 0; i < len(b.bA); i++ {
+		b.bA[i].RunOptimize()
+	}
+	b.runOptimized = true
+}
+
+func (b *BSI) HasRunCompression() bool {
+	return b.runOptimized
+}
+
+// Returns a pointer to existence bitmap
+func (b *BSI) GetExistenceBitmap() *Bitmap {
```

Comment:
Comment does not have the expected format.

Reply:
Done.